nuxeo-full-ps-template.yaml

```diff
@@ -0,0 +1,1989 @@
+                name: jenkins-configuration
+        dnsPolicy: ClusterFirst
+        restartPolicy: Always
+        schedulerName: default-scheduler
+        securityContext: {}
+        serviceAccount: jenkins
+        serviceAccountName: jenkins
+        terminationGracePeriodSeconds: 30
+        volumes:
+          - emptyDir: {}
+            name: jenkins-data
+          - configMap:
+              defaultMode: 420
+              items:
+                - key: MAVEN_SETTINGS
+                  path: settings.xml
+              name: ${APPLICATION_NAME}-maven-settings
+            name: jenkins-configuration
+    test: false
+    triggers:
+      - imageChangeParams:
+          automatic: true
+          containerNames:
+            - jenkins
+          from:
+            kind: ImageStreamTag
+            name: 'jenkins:latest'
+            namespace: openshift
+        type: ImageChange
+      - type: ConfigChange
+
+
+
+############################################################################################################
+# Commons
+############################################################################################################
+
+
+- apiVersion: v1
+  kind: Secret
+  metadata:
+    name: ${APPLICATION_NAME}-connect
+    labels:
+      app: ${APPLICATION_NAME}
+  stringData:
+    connect.properties: |-
+      NUXEO_STUDIO_PROJECT=${NUXEO_STUDIO_PROJECT}
+      NUXEO_STUDIO_PROJECT_VERSION=${NUXEO_STUDIO_VERSION}
+      NUXEO_CONNECT_USERNAME=${NUXEO_CONNECT_USERNAME}
+      NUXEO_CONNECT_PASSWORD=${NUXEO_CONNECT_PASSWORD}
+
+- apiVersion: v1
+  kind: Secret
+  type: kubernetes.io/basic-auth
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+      credential.sync.jenkins.openshift.io: 'true'
+    name: ${APPLICATION_NAME}-connect-username-password
+    annotations:
+      template.openshift.io/base64-expose-credentials: '{.data}'
+  stringData:
+    username: ${NUXEO_CONNECT_USERNAME}
+    password: ${NUXEO_CONNECT_PASSWORD}
+
+- apiVersion: v1
+  kind: Secret
+  type: kubernetes.io/ssh-auth
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+      credential.sync.jenkins.openshift.io: 'true'
+    name: ${APPLICATION_NAME}-github-ssh
+    annotations:
+      template.openshift.io/base64-expose-credentials: '{.data}'
+  stringData:
+    ssh-privatekey: |-
+      ${SSH_PRIVATEKEY}
+
+- apiVersion: v1
+  kind: Secret
+  type: Opaque
+  metadata:
+    labels:
+      app: ${APPLICATION_NAME}
+      credential.sync.jenkins.openshift.io: 'true'
+    name: ${APPLICATION_NAME}-instance-clid
+    annotations:
+      template.openshift.io/base64-expose-credentials: '{.data}'
+  stringData:
+    instance.clid: |-
+      ${INSTANCE_CLID}
+
+- apiVersion: v1
+  kind: ImageStream
+  metadata:
+    name: ${APPLICATION_NAME}-nuxeo
+    labels:
+      app: ${APPLICATION_NAME}
+  spec:
+    dockerImageRepository: nuxeo
+
+- apiVersion: v1
+  data:
+    MAVEN_SETTINGS: |-
```

Comment:
Current S2i build allows to use the Nexus HF repo by defining credentials, for instance when a project needs to be built against a HF. (cd https://github.com/nuxeo-sandbox/nuxeo-s2i/blob/master/s2i/bin/assemble#L34)

This is something that should be done here too.

Reply:
@dmetzler 
I would think that $NUXEO_HOTFIX_USERNAME would be the same as $NUXEO_CONNECT_USERNAME, no?

Reply:
done, with $NUXEO_CONNECT_USERNAME